ansible/requirements.yml

```diff
@@ -99,8 +99,9 @@
 - src: openmicroscopy.openstack-volume-storage
   version: 1.1.0
 
-- src: openmicroscopy.postgresql
-  version: 3.0.1
+- name: openmicroscopy.postgresql
```

Comment:
To avoid cascading dependency bumps the new role `src: ome.postgresql` should keep the same `name` for now